src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ClusterResourcesConfig.scala

```diff
@@ -1,14 +1,14 @@
 package org.broadinstitute.dsde.workbench.leonardo.config
 
-case class ClusterResourcesConfig(
-                                   configFolderPath: String,
-                                   initActionsScript: String,
-                                   clusterDockerCompose: String,
-                                   leonardoServicePem: String,
-                                   jupyterServerCrt: String,
-                                   jupyterServerKey: String,
-                                   jupyterRootCaPem: String,
-                                   jupyterRootCaKey: String,
-                                   jupyterProxySiteConf: String,
-                                   jupyterInstallExtensionScript: String
-                                 )
\ No newline at end of file
+import org.broadinstitute.dsde.workbench.leonardo.model.ClusterResource
+
+case class ClusterResourcesConfig(initActionsScript: ClusterResource,
```

Comment:
Super minor but is it necessary to a have `ClusterResource` case class, maybe a generic `FileName` would be better? idk which is superior honestly. Just musing.

Reply:
Well, I wanted this case class to specifically represent resources under `src/main/resources/jupyter` which should be uploaded to the cluster init bucket. Trying to distinguish between cluster "resources" (inside the leonardo.jar) and cluster "files" (on the filesystem of the leo docker).